doc/docs/master/how-tos/pipeline-operations/updating_pipelines.md

```diff
@@ -102,18 +102,20 @@ To update the code in your pipeline, complete the following steps:
 
    * If you prefer to use Pachyderm commands:
 
-     1. Run the following command:
+     1. Build your new image using `docker build`
```

Comment:
I'm not sure how much description is ideal here. Should we assume a user can follow up on building their docker image without further instruction?

Reply:
Added a link to the doc that explains how to build your image